mail_message_icon/views/mail_message_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8' ?>
+<odoo>
+  <record id="mail_message_view_form_mail_info" model="ir.ui.view">
```

Comment:
According to OCA Guidelines, id of view that inherit after view from another module should have exactly the same id as the original (module name will separate them apart) - [OCA Guidelines 2.5.3](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#253inherited-xml)
```suggestion
  <record id="mail_message_view_form" model="ir.ui.view">
```